### DISCUSSION TITLE
K1 Max back fan stays ON after print ends

### DISCUSSION THREAD

**Discussion by pavlicekdavid** [Guilouz/Creality-Helper-Script-Wiki#538] [Issues]
Hi, after I installed Fans Control Macros from Helper Script on my K1 Max and set up cooling settings in Orca like its in manual, back fan (fan1) always stays on at 82% after print and never turn off (wait all night). 

Does someone know how to repair it?
![Snímek obrazovky 2024-04-28 170943](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/130193202/53988dd4-782e-456a-8cc5-8a61c691500f)

Thank you.

**Comment by d3dorsett** [comment]
I have the same issue. The command M107 P1, should turn it off.

I have noticed in _gcode_macro.cfg_, the command is sent under [gcode_macro END_PRINT]. It appears to turn it off, then a few seconds later something else turns it back on and leaves it running.

[gcode_macro END_PRINT]
gcode:
  Qmode_exit
  EXCLUDE_OBJECT_RESET
  PRINT_PREPARE_CLEAR
  M220 S100
  SET_VELOCITY_LIMIT ACCEL=5000 ACCEL_TO_DECEL=2500
  TURN_OFF_HEATERS
  **M107 P1**
  M107 P2
  END_PRINT_POINT
  WAIT_TEMP_START
  M84

**Comment by buildjournal** [comment]
Bumping this up. @Guilouz 
I had the same issue for a few months now. Back fan stays on after print. Maybe around 5-10% power. Would like to know how to automatically turn off.
